diff --git a/node_modules/@aws-amplify/adapter-nextjs/dist/cjs/utils/createRunWithAmplifyServerContext.js b/node_modules/@aws-amplify/adapter-nextjs/dist/cjs/utils/createRunWithAmplifyServerContext.js
index 8f762d0..b0480d6 100644
--- a/node_modules/@aws-amplify/adapter-nextjs/dist/cjs/utils/createRunWithAmplifyServerContext.js
+++ b/node_modules/@aws-amplify/adapter-nextjs/dist/cjs/utils/createRunWithAmplifyServerContext.js
@@ -19,7 +19,6 @@ const createRunWithAmplifyServerContext = ({ config: resourcesConfig, tokenValid
         ...setCookieOptions,
         // enforced options when server-side auth is enabled
         ...(isServerSideAuthEnabled && {
-            ...constant_1.ENFORCED_SERVER_SIDE_AUTH_SET_COOKIE_OPTIONS,
             secure: isSSLOrigin,
         }),
         // only support root path
diff --git a/node_modules/@aws-amplify/adapter-nextjs/dist/esm/utils/createRunWithAmplifyServerContext.mjs b/node_modules/@aws-amplify/adapter-nextjs/dist/esm/utils/createRunWithAmplifyServerContext.mjs
index 57e7b6e..43c024e 100644
--- a/node_modules/@aws-amplify/adapter-nextjs/dist/esm/utils/createRunWithAmplifyServerContext.mjs
+++ b/node_modules/@aws-amplify/adapter-nextjs/dist/esm/utils/createRunWithAmplifyServerContext.mjs
@@ -16,7 +16,6 @@ const createRunWithAmplifyServerContext = ({ config: resourcesConfig, tokenValid
         ...setCookieOptions,
         // enforced options when server-side auth is enabled
         ...(isServerSideAuthEnabled && {
-            ...ENFORCED_SERVER_SIDE_AUTH_SET_COOKIE_OPTIONS,
             secure: isSSLOrigin,
         }),
         // only support root path
diff --git a/node_modules/@aws-amplify/adapter-nextjs/src/utils/createRunWithAmplifyServerContext.ts b/node_modules/@aws-amplify/adapter-nextjs/src/utils/createRunWithAmplifyServerContext.ts
index e574405..c8894e7 100644
--- a/node_modules/@aws-amplify/adapter-nextjs/src/utils/createRunWithAmplifyServerContext.ts
+++ b/node_modules/@aws-amplify/adapter-nextjs/src/utils/createRunWithAmplifyServerContext.ts
@@ -14,7 +14,6 @@ import {
 import { NextServer } from '../types';
 import {
 	DEFAULT_SERVER_SIDE_AUTH_SET_COOKIE_OPTIONS,
-	ENFORCED_SERVER_SIDE_AUTH_SET_COOKIE_OPTIONS,
 } from '../auth/constant';
 
 import { createCookieStorageAdapterFromNextServerContext } from './createCookieStorageAdapterFromNextServerContext';
@@ -39,7 +38,6 @@ export const createRunWithAmplifyServerContext = ({
 		...setCookieOptions,
 		// enforced options when server-side auth is enabled
 		...(isServerSideAuthEnabled && {
-			...ENFORCED_SERVER_SIDE_AUTH_SET_COOKIE_OPTIONS,
 			secure: isSSLOrigin,
 		}),
 		// only support root path
